.github/workflows/reduce-readme.yml

```diff
@@ -0,0 +1,34 @@
+name: Reduce README
+on:
+  push:
+    paths:
+    - '**'
+    branches: ['**']
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v3
+    - name: Install Asciidoctor Reducer
+      run: sudo gem install asciidoctor-reducer
+    - name: Reduce README
+      run: |
+        CURRENT_DIR="$( pwd -P)"
+        #Get a list of subdirectories that don't start with a .
+        subdirs=`find . -maxdepth 1 -type d ! -iname ".*"`
+        #Loop over the subdirectories
+        for subdir in $subdirs
```

Comment:
The redution examples didn't resolve all attributes, for instance {artifactId} (which could be simply the subdir value) coming from qs parent, or others defined locally in quickstart pom.xmls. I believe this is an acceptable limitation, but if you want to look for ways to solve this may be handy, it shows how to obtain property values from pom.xml and then inject it on another mvn command:

```
        echo "VERSION_BOMS=$(mvn -N org.apache.maven.plugins:maven-help-plugin:3.2.0:evaluate -Dexpression=project.version -q -DforceStdout)" >> $GITHUB_ENV
        mvn -U -B -fae clean package -Pprovisioned-server -Dversion.server.bom=${{ env.VERSION_BOMS }}

```